payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/HyperlinkedText.kt

```diff
@@ -65,9 +65,7 @@ fun HyperlinkedText(
 }
 
 private val urlPattern: Pattern = Pattern.compile(
-    "(?:^|[\\W])((ht|f)tp(s?):\\/\\/|www\\.)" +
-        "(([\\w\\-]+\\.){1,}?([\\w\\-.~]+\\/?)*" +
-        "[\\p{Alnum}.,%_=?&#\\-+()\\[\\]\\*$~@!:/{};']*)",
+    "(https?://[a-z0-9.-]+\\.[a-z]{2,3}(?:/\\S*?(?=\\.*(?:\\s|\$)))?)",
```

Comment:
Sorry for not bringing this up earlier, but can we use `android.util.Patterns.WEB_URL` instead of our own pattern?

Reply:
That patter will include the period at the end of an URL, which breaks the use case I want, where the URL is at the end of the error message followed by a period.

Reply:
Ah, makes sense. Should we add a comment with that info?

Reply:
Done